intellij/org.eclipse.xtext.xtext.idea.tests/.project

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought this created by gradle eclipse but probably I still don't have a clear picture on our config :)